frontend/src/v5/store/tickets/tickets.selectors.ts

```diff
@@ -109,3 +110,10 @@ export const selectTicketsByContainersAndFederations = createSelector(
 		return sortTicketsByCreationDate(tickets);
 	},
 );
+
+export const selectStatusConfigByTemplateId = createSelector(
+	(state) => state,
+	(state, modelId) => modelId,
+	(state, modelId, templateId) => templateId,
+	(storeState, modelId, templateId) => selectTemplateById(storeState, modelId, templateId)?.config?.status || DEFAULT_STATUS_CONFIG,
+);
```

Comment:
for reusing  selectTemplateById you can do this:
```suggestion
export const selectStatusConfigByTemplateId = createSelector(
	selectTemplateById,
	(template) => template?.config?.status || DEFAULT_STATUS_CONFIG,
);
```